### ISSUE TITLE
automatically update shop

### ISSUE THREAD

**Issue by mega2k0** [xyzz/acquisition#511]
i noticed that "Automatically update shop" funcion is not updating shop at forum, even if return no error.
as soon i push the "Update shop thread at forum" the items are appearing tho.

using the lastest build.

**Comment by DaneelTrevize** [comment]
Almost certainly because of ssiue #510

**Comment by xyzz** [comment]
No, I don't think so, the linked issue only affects the /login page.

**Comment by lab1992** [comment]
did you ever solve this Mega2k0?

**Comment by mega2k0** [comment]
fixed! probably api problem, not depending on our side
